### DISCUSSION TITLE
Broken: YTN (KOR), or Handling a number of streams on one YT channel

### DISCUSSION THREAD

**Discussion by arecki2** [iptv-org/community#470] [Show and tell]
### Broken Link

https://ythls.onrender.com/channel/UChlgI3UHCOnwUGzWzbJ3H5w.m3u8

### What happened to the stream?

Other

### Notes (optional)

As noted in the title, it's not that this stream is broken. It works fine :)

However, the issue (?) is that YTN YouTube channel often runs more than one stream at once, and I presume ythls.onrender.com/channel/* simply picks up the freshest one. I'm OK with that, but the effect is instead of regular YTN news output, I'm watching a special stream made of Typhoon Khanun reports submitted by viewers via KakaoTalk.

For comparison purposes:
Main stream is @ https://www.youtube.com/watch?v=FJfwehhzIhw so it would be https://ythls.onrender.com/video/FJfwehhzIhw.m3u8
and the footage one is https://www.youtube.com/watch?v=fAiHD0CLNms or https://ythls.onrender.com/video/fAiHD0CLNms.m3u8

There are many other YT channels that operate this way. But I'm afraid picking the right one for every instance or attaching them all into all the playlists here is beyond this project's scope 🤷 especially since every channel's rules are different; I know of one channel that normally changes streams twice a day, and still may add some webcams or whatnot in the meantime.

Just my 2 cents.

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#report-a-broken-stream)

**Comment by BellezaEmporium** [comment]
Not possible, as it relies on YouTube's API. The APIs made in such way that when you ask for "get_live_stream" it will always get you the one with most views, not the freshest one. Sadly, this also means that once you try getting the channel live and it has multiple live streams, it won't always work as expected.
